static/src/javascripts/projects/commercial/modules/prebid/bid-config.js

```diff
@@ -315,6 +315,92 @@ const getXaxisPlacementId = (sizes: PrebidSize[]): number => {
     return 13663304;
 };
 
+const getPangaeaPlacementId = (sizes: PrebidSize[]): number => {
+    type PangaeaSection = {
+        sections: Array<string>,
+        lb: number,
+        mmpu: number,
+        dmpu: number,
+    };
+    type PangaeaList = Array<PangaeaSection>;
+    const pangaeaList: PangaeaList = [
+        {
+            sections: ['business'],
+            lb: 13892359,
+            mmpu: 13892404,
+            dmpu: 13892360,
+        },
+        {
+            sections: ['culture'],
+            lb: 13892361,
+            mmpu: 13892405,
+            dmpu: 13892362,
+        },
+        {
+            sections: ['uk', 'us', 'au'],
+            lb: 13892363,
+            mmpu: 13892406,
+            dmpu: 13892364,
+        },
+        {
+            sections: ['news'],
+            lb: 13892365,
+            mmpu: 13892407,
+            dmpu: 13892366,
+        },
+        {
+            sections: ['money'],
+            lb: 13892367,
+            mmpu: 13892408,
+            dmpu: 13892368,
+        },
+        {
+            sections: ['sport'],
+            lb: 13892372,
+            mmpu: 13892410,
+            dmpu: 13892373,
+        },
+        {
+            sections: ['lifeandstyle', 'culture'],
+            lb: 13892411,
+            mmpu: 13892436,
+            dmpu: 13892437,
+        },
+        {
+            sections: ['technology', 'environment'],
+            lb: 13892376,
+            mmpu: 13892414,
+            dmpu: 13892377,
+        },
+        {
+            sections: ['travel'],
+            lb: 13892378,
+            mmpu: 13892415,
+            dmpu: 13892379,
+        },
+    ];
+
+    const section: string = config.page.section.toLowerCase();
```

Comment:
guardian eslint rule: `no-direct-access-config`

> 'Prefer accessing properties on config using get() method'

using `get()` has another benefit, you can pass a second parameter to account for if the property is undefined: `config.get('page.section', 'unknown')`

Reply:
In fact, I am doing something similar for AppNexus...
```js
const getAppNexusInvCode = (sizes: Array<PrebidSize>): ?string => {
    const device: string = getBreakpointKey();
    const section: string = config.get('page.section', 'unknown');
    const slotSize: PrebidSize | null = getLargestSize(sizes);
    if (slotSize) {
        return `${device}${section.toLowerCase()}${slotSize.join('x')}`;
    }
};
```
So we really should look at storing the section or config once and passing it into all the bid placement ID functions... This might be a codebase improvement target for a later PR 😅